src/reactionsystem.jl

```diff
@@ -279,6 +279,29 @@ function ReactionSystem(rxs::Vector{<:Reaction}, iv; kwargs...)
 end
 
 
+# Only used internally by the @reaction_network macro. Permits giving an initial order to the parameters, and then adds additional ones found in the reaction. Name could be changed.
+function make_ReactionSystem_internal(rxs::Vector{<:Reaction}, iv, no_sps::Nothing, ps_in; kwargs...)  

```

Comment:
Given the code redundancy, could you update the two argument constructor above to just call this too with an empty parameter vector?